Buildings/Experimental/DHC/Examples/Steam.mo

```diff
@@ -99,8 +95,7 @@ equation
     "Simulate and plot"),
```

Comment:
In plot 2, top plot:

`pla.port_bSerHea.P` is 4 bar and rest all `bld.port_b.p and plant.port_aSerHea.p` are in the range of 1-1.5bar.  By plotting all in 1 plot makes the 1 bar lines very closer to each other and we cannot see the difference. 

If the example has to show the different outlet pressures from each building, it will be good to plot the building pressures together.  Please let me know what you think.

 
![image](https://user-images.githubusercontent.com/77424734/162333873-eb3912e5-5068-4e50-a05f-38df606cf698.png)

Reply:
I agree splitting the pressures into 2 plots will be clearer. I've revised the script and will update the reference results as needed. Thanks!

Reply:
- [ ] Rerun unit tests & update results as needed